src/elements/netspeak-search.tsx

```diff
@@ -419,34 +440,41 @@ export class NetspeakSearch extends React.PureComponent<Props, State> {
 
 		return (
 			<div className="NetspeakSearch">
-				<div className="wrapper search-bar-wrapper">
-					<table>
-						<tbody>
-							<tr>
-								<td>
-									<NetspeakSearchBar
-										query={this.state.query}
-										onQueryEnter={this._onSearchBarQueryEnterHandler}
-										autoFocus={this.props.autoFocus}
-									/>
-								</td>
-								<td>
-									<TransparentButton
-										image={url(InfoImage)}
-										selected={this._areExamplesVisible()}
-										onClick={this._onExampleButtonClick}
-									/>
-								</td>
-								<td>
-									<TransparentButton image={url(ClearImage)} onClick={this._onClearButtonClick} />
-								</td>
-								{optional(!!this.props.history, () => (
-									<td>{this._renderHistoryPopup()}</td>
-								))}
-							</tr>
-						</tbody>
-					</table>
-				</div>
+				{optional(this.props.apiType === NetspeakApi.neural, () => (
```

Comment:
Controlling whether the search bar is interactable or not via the API kind is really hacky.